charts/scalardb/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for scalardb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount -- Default values for number of replicas.
+replicaCount: 1
+# nameOverride -- String to partially override scalardl.fullname template (will maintain the release name)
+nameOverride: ""
+# fullnameOverride -- String to fully override scalardl.fullname template
+fullnameOverride: ""
+
+scalardb:
+  # scalardb.contactPoints -- The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account.
+  contactPoints: cassandra
+  # scalardb.contactPort -- The database port number.
+  contactPort: 9042
+  # scalardb.username -- The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here.
+  username: cassandra
+  # scalardb.password -- The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here.
+  password: cassandra
+  # scalardb.storage -- Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set.
+  storage: cassandra
+
+  server:
+    # scalardb.server.port -- The port of Scalar DB server.
+    port: 60051
+    # scalardb.server.prometheus_http_endpoint_port -- The port of Prometheus service runnning in Scalar DB server.
+    prometheus_http_endpoint_port: 8080
+
+  image:
+    # scalardb.image.repository -- Docker image reposiory of Scalar DB server.
+    repository: ghcr.io/scalar-labs/scalardb-server
+    # scalardb.image.imagePullPolicy Specify a image pulling policy.
+    pullPolicy: IfNotPresent
+    # scalardb.image.tag -- Docker tag of the image.
+    tag: 3.1.0
+
+  # scalardb.imagePullSecrets -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+  imagePullSecrets: []
+
+strategy:
+  rollingUpdate:
+    # strategy.rollingUpdate -- The number of pods that can be created above the desired amount of pods during an update
+    maxSurge: 25%
+    # strategy.rollingUpdate -- The number of pods that can be unavailable during the update process
+    maxUnavailable: 25%
+  # strategy.type -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+  type: RollingUpdate
+
+service:
+  # service.type -- service types in kubernetes.
+  type: LoadBalancer
```

Comment:
Do we need to use `LoadBalancer` type in default ?

In envoy, defined the `LoadBalancer` type in custom config file.
https://github.com/scalar-labs/scalar-kubernetes/blob/64c3f6321eedffec681c4178850e7ddca9e317db/conf/scalardl-custom-values.yaml#L11

Reply:
I see. I'll change it back to ClusterIP